app/src/main/java/com/example/drawingactivity/DrawActivity.kt

```diff
@@ -125,6 +134,51 @@ class DrawActivity : AppCompatActivity() {
             if (it.layerList.isEmpty()) {
                 viewModel.addLayer(LayerViewModel(this.getString(R.string.layerHint, 1)))
             }
+            layoutList[0].isVisible = false
+            when (it.activeTool) {
```

Comment:
Same comment as in the other spot. This can be made much more elegant with better mapping

Reply:
talked in class